packages/blade/src/components/Accordion/__tests__/__snapshots__/Accordion.native.test.tsx.snap

```diff
@@ -82,7 +82,19 @@ exports[`<Accordion /> should render 1`] = `
               accessibilityRole="button"
               accessibilityState={
                 {
+                  "busy": undefined,
+                  "checked": undefined,
+                  "disabled": undefined,
                   "expanded": false,
+                  "selected": undefined,
```

Comment:
Did we make any changes to makeAccessible function? 

Only this accesibilityState seem to have been updating in snapshots. could be because of jest upgrade?